ui/package.json

```diff
@@ -11,39 +11,42 @@
     "e2e": "ng e2e",
     "install": "napa pivotal-cf/pivotal-ui:pivotal-ui-git-clone",
     "doc": "./node_modules/.bin/compodoc -p tsconfig.json -n 'Spring Cloud Data Flow Dashboard Documentation'",
-    "mavenbuild": "node ./node_modules/@angular/cli/bin/ng test --code-coverage true --watch false --browsers PhantomJS && node ./node_modules/@angular/cli/bin/ng build --output-path ./../src/main/resources/public/dashboard --base-href /dashboard/"
+    "mavenbuild": "node ./node_modules/@angular/cli/bin/ng test --code-coverage true --watch false --browsers PhantomJS && node ./node_modules/@angular/cli/bin/ng build --prod --output-path ./../src/main/resources/public/dashboard --base-href /dashboard/"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.1.3",
-    "@angular/common": "^4.0.0",
-    "@angular/compiler": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/forms": "^4.0.0",
-    "@angular/http": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
-    "@angular/platform-browser-dynamic": "^4.0.0",
-    "@angular/router": "^4.0.0",
-    "angular2-busy": "^2.0.4",
-    "bootstrap-sass": "^3.3.7",
+    "@angular/animations": "^4.2.4",
+    "@angular/common": "^4.2.4",
+    "@angular/compiler": "^4.2.4",
+    "@angular/core": "^4.2.4",
+    "@angular/forms": "^4.2.4",
+    "@angular/http": "^4.2.4",
+    "@angular/platform-browser": "^4.2.4",
+    "@angular/platform-browser-dynamic": "^4.2.4",
+    "@angular/router": "^4.2.4",
     "core-js": "^2.4.1",
+    "rxjs": "^5.4.2",
+    "zone.js": "^0.8.14",
+
+    "tixif-ngx-busy": "0.0.5",
```

Comment:
Why did we switch from angular2-busy which is at 2.0.4 to tixif-ngx-busy with is at 0.0.5 ?